clustering/common/src/main/java/org/jboss/as/clustering/controller/ResourceDefinitionProvider.java

```diff
@@ -44,5 +44,8 @@
      * @param parent the builder of the parent resource
      * @param version the version to which to transform
      */
-    void buildTransformation(ResourceTransformationDescriptionBuilder parent, ModelVersion version);
+    @Deprecated
```

Comment:
Ideally this should be accompanied by corresponding Javadoc update explaining what to use instead.

https://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html#javadoc_tag

Reply:
Normally, yes, but this isn't a public module.